pkgs/tools/admin/lxd/default.nix

```
@@ -78,14 +81,14 @@ buildGoModule rec {
   passthru.tests.lxd-ui = nixosTests.lxd-ui;
   passthru.ui = callPackage ./ui.nix { };
   passthru.updateScript = gitUpdater {
-    url = "https://github.com/lxc/lxd.git";
+    url = "https://github.com/canonical/lxd.git";
     rev-prefix = "lxd-";
   };
 
   meta = with lib; {
     description = "Daemon based on liblxc offering a REST API to manage containers";
-    homepage = "https://linuxcontainers.org/lxd/";
-    changelog = "https://github.com/lxc/lxd/releases/tag/lxd-${version}";
+    homepage = "https://documentation.ubuntu.com/lxd/en/latest";
```

```suggestion
    homepage = "https://ubuntu.com/lxd";
```
I just found out that lxd was moved to Canonical, which motivated the resignation of Graber https://stgraber.org/2023/07/10/time-to-move-on/